cmd/agent/daemon/daemon.go

```diff
@@ -93,6 +101,7 @@ func NewRunCommand(version string) *cobra.Command {
 	)
 
 	pflag.Var(&netflowGrouping, "netflow-grouping", "Group netflow to reduce cardinality. Eg: drop_src_port to drop source port")
+	pflag.Var(&ebpfEventsPolicy, "ebpf-events-policy", "Specify which ebpf events should be traced")
```

Comment:
Nice idea! Two things:
* `|` feels a bit weird as a separator, to be honest. I would go with `,`
* Can we expose this as a nice array in the helm chart values?

Reply:
Also not sure if it is worth it, but tracee allows for wildcards for events. This might be a useful feature to enable a certain class of events. Maybe a future improvement.